openfisca_france/param/param.xml

```diff
@@ -3149,7 +3149,9 @@
         <VALUE deb="1991-09-24" fin="2002-03-31" valeur="0" />
       </CODE>
       <CODE code="cpl6" description="Complément de catégorie 6 (majoration pour assistance d’une tierce personne invalide de 3e catégorie) " format="float" type="monetary">
-        <VALUE deb="2014-04-01" fuzzy="true" valeur="1096.50" />
+        <VALUE deb="2016-04-01" fuzzy="true" valeur="1104.18" />
+        <VALUE deb="2015-04-01" fin="2016-03-31" valeur="1103.08" />
+        <VALUE deb="2014-04-01" fin="2015-03-31" valeur="1096.50" /> <!-- Don't know the actual value -->
```

Comment:
May be [here](https://git.framasoft.org/french-tax-and-benefit-tables/ipp-tax-and-benefit-tables-yaml-clean/blob/master/baremes-ipp-prestations-sociales-social-benefits/aeeh.yaml)